virtme/commands/run.py

```diff
@@ -272,6 +280,11 @@ def make_parser() -> argparse.ArgumentParser:
     g.add_argument(
         "--force-9p", action="store_true", help="Use legacy 9p filesystem as rootfs"
     )
+    g.add_argument(
+        "--force-tcp",
```

Comment:
detail: should this be part of the `Remote Console` group below instead?

And maybe: `--force-ssh-tcp`?